bokeh/server/tornado.py

```diff
@@ -69,6 +73,7 @@ def __init__(self, applications,
                     route = p[0]
                 else:
                     route = key + p[0]
+                route = self._prefix + route
```

Comment:
I don't like the path either (well - I don't remember what it is but remember not liking it :-) I think I copied the old server here?)

I would think --prefix should "entirely contain" bokeh since it's presumably used to avoid conflicts with something else, I guess.